api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1232 @@
+    EventType and will be, if so desired, enforced by Nakadi. Usually Nakadi will respect the schema for
+    the EventTypes in accordance to how an owning Application defines them.
+    **Note:** *Currently the specification of the schema must be pushed into Nakadi on EventType creation;
+    in the future, assuming that Applications will expose the schema for its owned resources, Nakadi might
+    support fetching the schema directly from them.*
+    Nakadi might define globally minimal schema compliance requirements (i.e. some fields that must be
+    present on schemas of all EventTypes).
+
+    * The expected **validation** and **enrichment** procedures upon reception of an Event.
+    (*TODO: define validation and enrichment*)
+
+    * The **ordering** expectations of Events in this stream. (*TODO: define ordering not in terms of
+    partitioning*)
+
+    Except for defined enrichment rules, Nakadi will never manipulate the content of any Event.
+
+    Clients of Nakadi can be grouped in 2 categories: **Application owners** and **Clients** (clients in turn
+    are both **Producers** and **Consumers** of Events). Application owners interact with Nakadi via the
+    **Schema Registry API** for the definition of EventTypes, while Clients via the **Stream API** for
+    submission and reception of Events. A low level **Topic API** is available, providing full control and
+    responsibility of position tracking and partitioning (and therefore ordering) to the Clients.
+
+    The consumption of Events proceeds via the establishment of a named **Subscription** to an EventType.
+    Subscriptions are persistent relationships from an Application (which might have several instances) and
+    the stream of an EventType, whose consumption tracking is managed by Nakadi, freeing Consumers from any
+    responsibility in tracking the current position on a Stream.
+
+
+    Scope and status of this document
+    -------------------------------
+
+    The present API specification is in **draft** state and is subject to change.
+
+    In this document, ready for review are included:
+    * The Schema Registry API, including configuration possibilities for the Schema, Validation, Enrichment
+    and Partitioning of Events, and their effects on reception of Events.
+
+    * The stantardised event format (see definition of Event, BusinessEvent and DataChangeEvent) (Note: at a
+    later moment this will be configurable and not be inherent part of this API).
+
+    * Topic API (low level).
+
+    Other aspects of the Event Bus are at this moment to be defined and otherwise specificied, not included
+    in this version of this specification.
+
+    Notable omissions here are:
+    * The contract between Nakadi and clients in the context of the high-level API (i.e. the process for
+    clients to establish subscriptions).
+
+    * Submission of events to topics in batched manner.
+
+    * Security scopes (OAuth) for the different operations
```

Comment:
Why not? It's very basic and I would not expect exotic event api access rights, simply:
<event bus appl id>.read and <event bus appl id>.write